### ISSUE TITLE
NZBGET_userscript.sh not installing correctly

### ISSUE THREAD

**Issue by tac00** [kylerw/MiscScripts#2]
When I run the script inside the Radarr container from Linuxserver.io it dosen't install all the dependencies and I keep getting errors. I am just supposed to run the script inside the container without changing anything in it?

**Comment by tac00** [comment]
I have also tried to install the Alpine script in Deluge docker image from Linuxserver.io. And I keep getting this error regarding pip:

`
Traceback (most recent call last):
  File "/usr/bin/pip", line 11, in <module>
    load_entry_point('pip==9.0.1', 'console_scripts', 'pip')()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 560, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2642, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2296, in load
    return self.resolve()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2302, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named pip`

**Comment by tac00** [comment]
Okay it seems like I got everything working. 

I can post my modified script for Deluge if you want it?

**Comment by kylerw** [comment]
Yeah, I'll post it up here for others if you're okay with that.

**Comment by tac00** [comment]
@kylerw I will post it up here later. When you have the mp4 automator installed inside your download container do Couchpotato and Sonarr handle your renaming of the converted file?

**Comment by kylerw** [comment]
@tac00 correct. The file is converted and then sent to CP/Sonarr to run their process which will rename and move if that is configured in the settings.

**Comment by tac00** [comment]
@kylerw  Okay now I understand. Should complete download handling and movie naming be enabled in Sonarr for this to work?  Im confused about the settings in the download handlers like CP and Sonarr.

**Comment by kylerw** [comment]
Yeah, if only installing for NZBGet or Duluge you just set it up there according to mp4a instructions and keep sonarr/others the same. If you want rename/move, make sure they are setup.

Example of my NZBGet settings:

![image](https://cloud.githubusercontent.com/assets/370860/22554572/d05fd1fa-e91d-11e6-9337-83758e605fe2.png)

**Comment by tac00** [comment]
I got Deluge and Radarr working together with mp4a. 

But I think I will install mp4a inside my Radarr and Sonarr containers and then run postRadarr and postSonarr instead because when using the download container to convert the converted mp4 fil will not be tagged correctly and art work is not included in the file.

**Comment by Fma965** [comment]
are you able to post your modified script for radarr / deluge?
